diff of execution of:
echo '(cat infile | grep in) > outfile; cat outfile' | tr ';' '\n' | [shell]
--- expected	2024-09-12 10:36:51.354066161 +0200
+++ found	2024-09-12 10:36:51.370066251 +0200
@@ -1,7 +1,7 @@
 infile content
 infile second line
 
 ======== stderr =======
-
+[progname]: outfile: No such file or directory
 ======== return =======
 return value: 0
